src/components/DragGroup/DragGroup.js

```diff
@@ -157,9 +157,9 @@ class DragGroup extends React.Component {
 
   renderTitle = expanded =>
     this.props.allowDrag ? (
-      <DragGroupTitle expanded={expanded} onDelete={this.props.onRemoved}>
+      <Title expanded={expanded} onDelete={this.props.onRemoved}>
```

Comment:
Nice, thanks for the bug fix! Doesn't `Title` need to be referenced as `this.props.Title` to work, though?

Reply:
Oops.. yes, actually wanted test this code before PR, but couldn't run its local.